new-site/src/docs/components/index.mdx

```diff
@@ -0,0 +1,8 @@
+---
+slug: "/components"
+title: "Components overview"
+---
+
+# Components overview
+
+Some component content
```

Comment:
I think we could add the component table here as we have it on old site

Reply:
I have been dreaming of a list of component cards. They would include a small image of the component, the component name and maybe the status. Then you could filter that list of cards with an input.

I guess for now we can just add the table too.

Reply:
Something like this; maybe with an image and some relevant tags.

<img width="1222" alt="image" src="https://user-images.githubusercontent.com/4214671/162198516-547b6688-9c82-4198-a2da-10d6681fcfdc.png">

Reply:
Ok 👍 Should we make a separate task about that?

Reply:
>  Should we make a separate task about that?
I will add a ticket and add basic specs.